src/client/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -26,11 +27,11 @@ internal class InteractiveRequest : RequestBase
 
         private readonly SortedSet<string> _extraScopesToConsent;
         private readonly IWebUI _webUi;
-        private AuthorizationResult _authorizationResult;
         private string _codeVerifier;
         private string _state;
         private readonly AcquireTokenInteractiveParameters _interactiveParameters;
         private MsalTokenResponse _msalTokenResponse;
+        internal AuthorizationResult AuthResult { get; private set; }
```

Comment:
Let's keep this private. If needed, InteractiveRequest can pass it in the constructor of BrokerInteractiveRequest.